tox.ini

```diff
@@ -43,7 +43,7 @@ commands = coverage erase
 
 [testenv:build_docs]
 changedir = {toxinidir}
-extras = docs
+deps = -r{toxinidir}/requirements.txt
```

Comment:
If we wanted to have a separate lockfile for the doc build, we could put it at `docs/requirements.txt` instead.